Minutor won't compile with gcc 10.1.0.

This is the returned error
```
usr/bin/ld: properties.o: in function `Properties::Properties(QWidget*)':
properties.cpp:(.text+0x37d): undefined reference to `PropertieTreeCreator::PropertieTreeCreator()'
collect2: error: ld returned 1 exit status
```
The first faulty commit is ac6ef8a4ce4eecae9de677d32b8946f9e931aec1
which indeed modifies properties.cpp and propertietreecreator.cpp.

If other information is required just ask me, I'll be happy to provide it.

--- a/propertietreecreator.cpp
+++ b/propertietreecreator.cpp
@@ -8,47 +8,6 @@ PropertieTreeCreator::PropertieTreeCreator() {
   summary.insert("Attributes[]", "{Name} = {Base}");
 }
 
-
-template <class IterableT>
-void PropertieTreeCreator::ParseIterable(QTreeWidgetItem* node, const IterableT& seq) {
-  typename IterableT::const_iterator it, itEnd = seq.end();
-  for (it = seq.begin(); it != itEnd; ++it) {
-    QTreeWidgetItem* child = new QTreeWidgetItem();
-    child->setData(0, Qt::DisplayRole, it.key());
-    child->setData(1, Qt::DisplayRole, GetSummary(it.key(), it.value()));
-    CreateTree(child, it.value());
-    node->addChild(child);
-  }
-}
-
-
-template <class IterableT>
-void PropertieTreeCreator::ParseList(QTreeWidgetItem* node, const IterableT& seq) {
-  typename IterableT::const_iterator it, itEnd = seq.end();
-  int i = 0;
-  // skip 1 sized arrays
-  if (seq.size() == 0) {
-    // empty
-    if (node)
-      node->setData(1, Qt::DisplayRole, "<empty>");
-  } else if (seq.size() == 1) {
-    CreateTree(node, seq.first());
-    if (node)
-      node->setData(1, Qt::DisplayRole, GetSummary("[0]", seq.first()));
-  } else {
-    for (it = seq.begin(); it != itEnd; ++it) {
-      QTreeWidgetItem* child = new QTreeWidgetItem();
-      QString key = QString("[%1]").arg(i++);
-      child->setData(0, Qt::DisplayRole, key);
-      child->setData(1, Qt::DisplayRole, GetSummary(key, *it));
-      CreateTree(child, *it);
-
-      node->addChild(child);
-    }
-  }
-}
-
-
 void PropertieTreeCreator::CreateTree(QTreeWidgetItem* node, const QVariant& v) {
   switch (v.type()) {
     case QMetaType::QVariantMap:

--- a/propertietreecreator.h
+++ b/propertietreecreator.h
@@ -4,6 +4,7 @@
 #include <QString>
 #include <QMap>
 #include <QVariant>
+#include <QTreeWidgetItem>
 
 class QTreeWidgetItem;
 
@@ -23,5 +24,46 @@ public:
 private:
   QMap<QString, QString> summary;
 };
+
+
+template <class IterableT>
+inline void PropertieTreeCreator::ParseIterable(QTreeWidgetItem* node, const IterableT& seq) {
+  typename IterableT::const_iterator it, itEnd = seq.end();
+  for (it = seq.begin(); it != itEnd; ++it) {
+    QTreeWidgetItem* child = new QTreeWidgetItem();
+    child->setData(0, Qt::DisplayRole, it.key());
+    child->setData(1, Qt::DisplayRole, GetSummary(it.key(), it.value()));
+    CreateTree(child, it.value());
+    node->addChild(child);
+  }
+}
+
+
+template <class IterableT>
+inline void PropertieTreeCreator::ParseList(QTreeWidgetItem* node, const IterableT& seq) {
+  typename IterableT::const_iterator it, itEnd = seq.end();
+  int i = 0;
+  // skip 1 sized arrays
+  if (seq.size() == 0) {
+    // empty
+    if (node)
+      node->setData(1, Qt::DisplayRole, "<empty>");
+  } else if (seq.size() == 1) {
+    CreateTree(node, seq.first());
+    if (node)
+      node->setData(1, Qt::DisplayRole, GetSummary("[0]", seq.first()));
+  } else {
+    for (it = seq.begin(); it != itEnd; ++it) {
+      QTreeWidgetItem* child = new QTreeWidgetItem();
+      QString key = QString("[%1]").arg(i++);
+      child->setData(0, Qt::DisplayRole, key);
+      child->setData(1, Qt::DisplayRole, GetSummary(key, *it));
+      CreateTree(child, *it);
+
+      node->addChild(child);
+    }
+  }
+}
+
 
 #endif // PROPERTIETREECREATOR_H

